cadctap/cadctap/tests/test_core.py

```diff
@@ -89,15 +90,26 @@
 
 THIS_DIR = os.path.dirname(os.path.realpath(__file__))
 TESTDATA_DIR = os.path.join(THIS_DIR, 'data')
+BASE_URL = 'https://ws-cadc.canfar.net/youcat/availability'
 
 
 class MyExitError(Exception):
     def __init__(self):
         self.message = "MyExitError"
 
 
-@patch('cadctap.core.net.BaseWsClient.put')
-def test_create_table(base_put_mock):
+@patch('netrc.netrc')
+def test_get_subject_from_netrc(netrc_mock):
+    netrc_instance = netrc_mock.return_value
+    netrc_instance.hosts = {'no_such_host': 'my.host.ca'}
+    subject = _get_subject_from_netrc()
+    assert(subject is None)
+
```

Comment:
Can you add the case where a netrc subject is created? And the case where the cert subject is created? That would also bump up the coverage.

Reply:
Added.